fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -783,19 +784,23 @@ def load_local_state_dict(
     @contextlib.contextmanager
     def no_sync(self) -> Generator:
```

Comment:
this naming might be canonical and this comment might be out of scope, but `no_grad_sync` would be a clearer name.

Reply:
I think this name comes from DDP, which has this no_sync context. I think we want to keep FSDP and DDP the same. I can certainly change it if people think otherwise.